### ISSUE TITLE
[Question] Is there other way to use keyd without Xcompose ? 

### ISSUE THREAD

**Issue by barcell1** [rvaiya/keyd#901]
Im using Keyd and it works as expected in terms of remapping BUT i fell a lag when keyd do the remap. I see the original input and the change input happening in my screen, i think im using the wrong backend.

**Comment by barcell1** [comment]
thanks, and sorry if i misusderstanding , what i did, according to keyd was :
i edit the /etc/keyd/ file in there according to my needs and keyd monitor and :

"ln -s /usr/share/keyd/keyd.compose ~/.XCompose"
"ibus restart"

Is there another way to do it ? you could point out to me please ?

**Comment by nsbgn** [comment]
No, that is the way for the compose mechanism. (There are other ways, but they come down to the same thing.)

The alternative way to enter special symbols is to set your XKB layout (that is, outside of keyd!) to whatever is most appropriate for you. (This may be a custom XKB layout, too.) You do this using `setxkblayout` on X11, or the appropriate method for your Wayland compositor.

You would then press the key or the sequence of keys needed to type the symbol you're after, and check `keyd monitor` to see the corresponding name within `keyd`. Then you can use those names in your config.

For example, if in your layout `rightalt` and `5` emits €, you can use `G-5` to get €.

Both methods have tradeoffs. It's good to remember that the names keyd gives to keycodes are just for convenience, and the relation to the symbols appearing on your screen is only established at a later stage through XKB.

**Comment by barcell1** [comment]
> For example, if in your layout `rightalt` and `5` emits €, you can use `G-5` to get €.

i thinking im starting to get what you are saying, so suppose i want € and my current layout do that with `rightalt` and `5` , so  i can use `G-5` but i want to change to  'shift' and '5' , i could set the keyd .conf file under /etc/keyd/ to  :

[shift]
5 = G-5 

this would be more performant than using .Xcompose
